wiremock/mappings/HmppsAuthAndDelius_GetUser_LaoFullAccess.json

```diff
@@ -0,0 +1,23 @@
+{
+  "priority": 2,
+  "request": {
+    "method": "GET",
+    "urlPattern": "/user/LAOFULLACCESS"
```

Comment:
Probably makes sense to make this case-insensitive e.g 

`(?i)\/user\/LAOFULLACCESS`

See https://github.com/ministryofjustice/hmpps-approved-premises-tools/pull/75

Reply:
That's great, thank you. Done